python/paddle/fluid/contrib/layers/nn.py

```diff
@@ -427,3 +424,71 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def fused_embedding_seq_pool(input,
+                             size,
+                             is_sparse=False,
+                             padding_idx=None,
+                             combiner='sum',
+                             param_attr=None,
+                             dtype='float32'):
+    """
+    **Embedding Sequence pool**
+
+    This layer is used to lookup and pool embeddings of IDs, provided by :attr:`input`, in
+    a lookup table. The result of this lookup is the embedding of each ID in the :attr:`input`.
+    The combiner type is mentioned in the Args.
+
+    Args:
+        input (Variable): Input is a Tensor<int64> Variable, which contains the IDs information.
+            The value of the input IDs should satisfy :math:`0<= id < size[0]`.
+        size (tuple|list): The shape of the look up table parameter. It should
+            have two elements which indicate the size of the dictionary of
+            embeddings and the size of each embedding vector respectively.
+        is_sparse (bool): The flag indicating whether to use sparse update.
+            Default: False.
+        padding_idx (int|long|None): It will output all-zero padding data whenever
```

Comment:
"It will output all-zero padding data whenever lookup encounters". I don't get your point

Reply:
the output will be all-zero if a ID in the inputs is equal padding_idx